modules/web/js/ballerina/diagram2/views/default/components/decorators/panel-decorator.css

```diff
@@ -52,4 +52,10 @@
 
 .protocol-rect-http{
     fill: #3498db !important;
+}
+.publicPrivateRectHolder{
```

Comment:
Better to have class names like public-private-rect-holder since we have followed that through out the project.